Dockerfile

```
@@ -1,19 +1,19 @@
-# Base image with python3.9 and enabled powertools and epel repo
+# Base image with python3.11 and enabled powertools and epel repo
 ARG BASE_IMAGE=quay.io/centos/centos:stream8
 FROM $BASE_IMAGE as base
 
 COPY misp-enable-epel.sh /usr/bin/
 RUN echo "tsflags=nodocs" >> /etc/yum.conf && \
     dnf update -y --setopt=install_weak_deps=False && \
-    dnf install -y python39 dnf-plugins-core && \
-    alternatives --set python3 /usr/bin/python3.9 && \
+    dnf install -y python3.11 dnf-plugins-core && \
```

pip is missing
```
python-build 2/2] RUN --mount=type=tmpfs,target=/tmp mkdir /tmp/source &&     cd /tmp/source &&     git config --system http.sslVersion tlsv1.3 &&     COMMIT=$(git ls-remote https://github.com/MISP/misp-modules.git v2.4.176 | cut -f1) &&     curl --proto '=https' --tlsv1.3 --fail -sSL https://github.com/MISP/misp-modules/archive/$COMMIT.tar.gz | tar zx --strip-components=1 &&     pip3 --no-cache-dir wheel --wheel-dir /wheels -r REQUIREMENTS &&     echo $COMMIT > /misp-modules-commit:
1.910 /bin/sh: pip3: command not found

[stage-2 4/5] RUN pip3 --no-cache-dir install --no-warn-script-location --user /wheels/* sentry-sdk==1.5.1 &&     echo "__all__ = ['cache', 'sentry']" > /home/misp-modules/.local/lib/python3.11/site-packages/misp_modules/helpers/__init__.py &&     chmod -R u-w /home/misp-modules/.local/:```
0.134 /bin/sh: pip3: command not found
```

```suggestion
    dnf install -y python3.11 python3.11-pip dnf-plugins-core && \
```

---

Adding python3.11-pip worked for me.